appcompat/appcompat-resources/src/main/java/androidx/appcompat/widget/VectorEnabledTintResources.java

```diff
@@ -38,16 +37,6 @@
 public class VectorEnabledTintResources extends ResourcesWrapper {
     private static boolean sCompatVectorFromResourcesEnabled = false;
```

Comment:
Probably this can be removed too (and the corresponding methods using too), but I wasn't sure if it was ok to go that far.